ArchivesPortalEuropeFoundation/Topic-Detection
# Dockerise the webapp

@kstamatis This is still a work in progress, but whenever you have a moment would you or someone from the team be able to give it a quick test? (tagging @kerstarno so you are also in the loop)

I think now if someone clones the repo and switches to the `add-docker` branch, by running:

```
docker build --tag ape .
docker run -p 5000:5000 ape
```

should start the web-app in the container and it will communicate through port 5000.

To test it:
```
curl -v 127.0.0.1:5000/query\?lang=fr\&type=concept\&text=et\&n_res=10\&broad_entity_search=False\&boolean_search=False
```
This should give you back a html table as a response with some entries in French.

If this works fine I just need to work a bit on the Volumes part to add the large datasets (for the moment the web-app works in test-mode using only samples which are already in the repo because they are very light).

Closes #55
